test/dynamo/test_dynamo.py

```diff
@@ -374,6 +375,40 @@ def test_resnet18(self, initialize_on_cuda):
     self.assertEqual(
         met.metric_data('RunCachedGraphOutputData')[0], sample_count)
 
+  @skipOnTpu
```

Comment:
don't skip on TPU, just don't do the `allclose` check, you can still check the counter etc.

Reply:
Updated

Reply:
the test is still marked with `@skipontpu`